features/security-monitoring.md

```diff
@@ -0,0 +1,30 @@
+# Security Monitoring
+The Private Packagist security monitoring feature searches the dependencies of your private packages for known security vulnerabilities.
+For Private Packagist to be able to monitor your project the project needs to be added as a package in Private Packagist even if you don't intend to install it.
```

Comment:
Can we highlight this somehow? Maybe use quote? Also rephrase please:

"Security monitoring is available for packages in your organization. To monitor your projects, you must add them to Private Packagist as packages, even if you do not intend to install them as a dependency."

Reply:
I take it highlighting this similar to how we show a warning no the bitbucket enterprise install guide is too strong: https://packagist.com/docs/enterprise/bitbucket-integration-setup

Reply:
I think that is fine, but instead of above the headline, display it above the secondary headline "Configuring Monitoring Settings"?

Reply:
As an alternative we can also you a block quote which is not as strong
![Screenshot 2020-03-27 at 15 37 55](https://user-images.githubusercontent.com/442056/77773043-f6db3380-7040-11ea-9dae-fe5707561941.png)

Reply:
Eh I think in that case rather the big warning sign

Reply:
Ok, added right above the headline "Configuring Monitoring Settings"